examples/react18/src/App.tsx

```diff
@@ -15,8 +15,13 @@ const handleFailRequestsOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
   state.failRequests = e.target.checked;
 };
 
+if (!process.env.REACT_APP_CLIENT_SECRET) {
+  console.error('REACT_APP_CLIENT_SECRET not set in .env');
+  process.exit(1);
+}
+
 const confidence = Confidence.create({
-  clientSecret: 'RxDVTrXvc6op1XxiQ4OaR31dKbJ39aYV',
+  clientSecret: process.env.REACT_APP_CLIENT_SECRET,
```

Comment:
I suggest aligning on the same environment variable for all example apps.

Reply:
React picks up only env variables with REACT_APP at the front, so we would be using the REACT_APP one, which can be confusing

Reply:
That is really confusing! :shaking-fist: